### ISSUE TITLE
Issue with caliban package with installing using pip

### ISSUE THREAD

**Issue by hamzaziizzz** [google/caliban#104]
There is an while installing the caliban package using pip. The error says **python setup.py egg_info did not run successfully.** I am attaching the command line output here. 
```
Collecting caliban
  Using cached caliban-0.4.1-py3-none-any.whl (157 kB)
Collecting absl-py
  Using cached absl_py-1.2.0-py3-none-any.whl (123 kB)
Collecting google-auth>=1.19.0
  Using cached google_auth-2.11.0-py2.py3-none-any.whl (167 kB)
Collecting google-cloud-container>=0.3.0
  Downloading google_cloud_container-2.11.2-py2.py3-none-any.whl (202 kB)
     ---------------------------------------- 202.8/202.8 kB 2.4 MB/s eta 0:00:00
Collecting lark-parser<0.8.0,>=0.7.1
  Downloading lark-parser-0.7.8.tar.gz (276 kB)
     ---------------------------------------- 276.2/276.2 kB 2.8 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Collecting blessings
  Downloading blessings-1.7-py3-none-any.whl (18 kB)
Collecting yaspin>=0.16.0
  Using cached yaspin-2.2.0-py3-none-any.whl (18 kB)
Collecting kubernetes>=10.0.1
  Using cached kubernetes-24.2.0-py2.py3-none-any.whl (1.5 MB)
Collecting commentjson
  Downloading commentjson-0.9.0.tar.gz (8.7 kB)
  Preparing metadata (setup.py) ... done
Collecting urllib3>=1.25.7
  Downloading urllib3-1.26.12-py2.py3-none-any.whl (140 kB)
     ---------------------------------------- 140.4/140.4 kB 8.1 MB/s eta 0:00:00
Collecting psycopg2-binary==2.8.5
  Using cached psycopg2-binary-2.8.5.tar.gz (381 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [25 lines of output]
      C:\Users\Hamza Aziz\AppData\Local\Programs\Python\Python310\lib\site-packages\setuptools\config\setupcfg.py:463: SetuptoolsDeprecationWarning: The license_file parameter is deprecated, use license_files instead.
        warnings.warn(msg, warning_class)
      running egg_info
      creating C:\Users\Hamza Aziz\AppData\Local\Temp\pip-pip-egg-info-1p1u8oyy\psycopg2_binary.egg-info
      writing C:\Users\Hamza Aziz\AppData\Local\Temp\pip-pip-egg-info-1p1u8oyy\psycopg2_binary.egg-info\PKG-INFO
      writing dependency_links to C:\Users\Hamza Aziz\AppData\Local\Temp\pip-pip-egg-info-1p1u8oyy\psycopg2_binary.egg-info\dependency_links.txt
      writing top-level names to C:\Users\Hamza Aziz\AppData\Local\Temp\pip-pip-egg-info-1p1u8oyy\psycopg2_binary.egg-info\top_level.txt
      writing manifest file 'C:\Users\Hamza Aziz\AppData\Local\Temp\pip-pip-egg-info-1p1u8oyy\psycopg2_binary.egg-info\SOURCES.txt'

      Error: pg_config executable not found.

      pg_config is required to build psycopg2 from source.  Please add the directory
      containing pg_config to the $PATH or specify the full executable path with the
      option:

          python setup.py build_ext --pg-config /path/to/pg_config build ...

      or with the pg_config option in 'setup.cfg'.

      If you prefer to avoid building psycopg2 from source, please install the PyPI
      'psycopg2-binary' package instead.

      For further information please check the 'doc/src/install.rst' file (also at
      <https://www.psycopg.org/docs/install.html>).

      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by sagnik1511** [comment]
Hi, did you try running this command `pip install psycopg2-binary`? That might help.

**Comment by hamzaziizzz** [comment]
I have tried this as well. I can't understand what is the error. 
```
error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [23 lines of output]
      running egg_info
      creating /tmp/pip-pip-egg-info-dgnbazb8/psycopg2_binary.egg-info
      writing /tmp/pip-pip-egg-info-dgnbazb8/psycopg2_binary.egg-info/PKG-INFO
      writing dependency_links to /tmp/pip-pip-egg-info-dgnbazb8/psycopg2_binary.egg-info/dependency_links.txt
      writing top-level names to /tmp/pip-pip-egg-info-dgnbazb8/psycopg2_binary.egg-info/top_level.txt
      writing manifest file '/tmp/pip-pip-egg-info-dgnbazb8/psycopg2_binary.egg-info/SOURCES.txt'
      
      Error: pg_config executable not found.
      
      pg_config is required to build psycopg2 from source.  Please add the directory
      containing pg_config to the $PATH or specify the full executable path with the
      option:
      
          python setup.py build_ext --pg-config /path/to/pg_config build ...
      
      or with the pg_config option in 'setup.cfg'.
      
      If you prefer to avoid building psycopg2 from source, please install the PyPI
      'psycopg2-binary' package instead.
      
      For further information please check the 'doc/src/install.rst' file (also at
      <https://www.psycopg.org/docs/install.html>).
      
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by sritchie** [comment]
Closing, as this should be fixed with the `psycopg_binary` upgrade from #109.
